blocks/quote/quote.js

```diff
@@ -0,0 +1,29 @@
+import { decorateIcons } from '../../scripts/aem.js';
+import { span, ul, li } from '../../scripts/dom-helpers.js';
+
+export default function decorate(block) {
+  const list = ul(...[...block.children].map((row) => li(...[...row.children].map((divElement) => {
+    if (
+      divElement.children.length === 1
+      && divElement.querySelector('picture')
+    ) {
+      divElement.className = 'quote-block-image';
+    } else divElement.className = 'quote-block-body';
+
+    return divElement;
+  }))));
+
+  block.textContent = '';
+  block.append(list);
```

Comment:
How about this?
```suggestion
  block.replaceWith(list);
```

Reply:
I should probably also add all attributes of the block to the list then

Reply:
fair enough

Reply:
replaceChildren tends to work well in those cases, but this is good